{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "python-gnupg 0.4.3 allows context-dependent attackers to trick gnupg to decrypt other ciphertext than intended. To perform the attack, the passphrase to gnupg must be controlled by the adversary and the ciphertext should be trusted. Related to a \"CWE-20: Improper Input Validation\" issue affecting the affect functionality component."
                }
            ],
            "problemtypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-12-15",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "106756",
                    "url": "http://www.securityfocus.com/bid/106756"
                },
                {
                    "name": "http://packetstormsecurity.com/files/151341/Python-GnuPG-0.4.3-Improper-Input-Validation.html",
                    "url": "http://packetstormsecurity.com/files/151341/Python-GnuPG-0.4.3-Improper-Input-Validation.html"
                },
                {
                    "name": "https://pypi.org/project/python-gnupg/#history",
                    "url": "https://pypi.org/project/python-gnupg/#history"
                },
                {
                    "name": "SU-2019:0143-1",
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2019-02/msg00008.html"
                },
                {
                    "name": "SUSE-SU-2019:0239-1",
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2019-02/msg00058.html"
                },
                {
                    "name": "[SECURITY] [DLA 1675-1] 20190214 python-gnupg security update",
                    "url": "https://lists.debian.org/debian-lts-announce/2019/02/msg00021.html"
                },
                {
                    "name": "20190125 CVE-2019-6690: Improper Input Validation in python-gnupg",
                    "url": "https://seclists.org/bugtraq/2019/Jan/41"
                },
                {
                    "name": "https://blog.hackeriet.no/cve-2019-6690-python-gnupg-vulnerability/",
                    "url": "https://blog.hackeriet.no/cve-2019-6690-python-gnupg-vulnerability/"
                },
                {
                    "name": "USN-3964-1",
                    "url": "https://usn.ubuntu.com/3964-1/"
                },
                {
                    "name": "FEDORA-2019-06f5bbdaf5",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/W6KYZMN2PWXY4ENZVJUVTGFBVYEVY7II/"
                },
                {
                    "name": "FEDORA-2020-17fb3273b2",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/X4VFRUG56542LTYK4444TPJBGR57MT25/"
                },
                {
                    "name": "FEDORA-2020-e67d007a67",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3WMV6XNPPL3VB3RQRFFOBCJ3AGWC4K47/"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2019-6690",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "python-gnupg 0.4.3 allows context-dependent attackers to trick gnupg to decrypt other ciphertext than intended. To perform the attack, the passphrase to gnupg must be controlled by the adversary and the ciphertext should be trusted. Related to a \"CWE-20: Improper Input Validation\" issue affecting the affect functionality component."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "106756",
                            "refsource": "BID",
                            "url": "http://www.securityfocus.com/bid/106756"
                        },
                        {
                            "name": "http://packetstormsecurity.com/files/151341/Python-GnuPG-0.4.3-Improper-Input-Validation.html",
                            "refsource": "MISC",
                            "url": "http://packetstormsecurity.com/files/151341/Python-GnuPG-0.4.3-Improper-Input-Validation.html"
                        },
                        {
                            "name": "https://pypi.org/project/python-gnupg/#history",
                            "refsource": "MISC",
                            "url": "https://pypi.org/project/python-gnupg/#history"
                        },
                        {
                            "name": "SU-2019:0143-1",
                            "refsource": "SUSE",
                            "url": "http://lists.opensuse.org/opensuse-security-announce/2019-02/msg00008.html"
                        },
                        {
                            "name": "SUSE-SU-2019:0239-1",
                            "refsource": "SUSE",
                            "url": "http://lists.opensuse.org/opensuse-security-announce/2019-02/msg00058.html"
                        },
                        {
                            "name": "[SECURITY] [DLA 1675-1] 20190214 python-gnupg security update",
                            "refsource": "MLIST",
                            "url": "https://lists.debian.org/debian-lts-announce/2019/02/msg00021.html"
                        },
                        {
                            "name": "20190125 CVE-2019-6690: Improper Input Validation in python-gnupg",
                            "refsource": "BUGTRAQ",
                            "url": "https://seclists.org/bugtraq/2019/Jan/41"
                        },
                        {
                            "name": "https://blog.hackeriet.no/cve-2019-6690-python-gnupg-vulnerability/",
                            "refsource": "MISC",
                            "url": "https://blog.hackeriet.no/cve-2019-6690-python-gnupg-vulnerability/"
                        },
                        {
                            "name": "USN-3964-1",
                            "refsource": "UBUNTU",
                            "url": "https://usn.ubuntu.com/3964-1/"
                        },
                        {
                            "name": "FEDORA-2019-06f5bbdaf5",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/W6KYZMN2PWXY4ENZVJUVTGFBVYEVY7II/"
                        },
                        {
                            "name": "FEDORA-2020-17fb3273b2",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/X4VFRUG56542LTYK4444TPJBGR57MT25/"
                        },
                        {
                            "name": "FEDORA-2020-e67d007a67",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3WMV6XNPPL3VB3RQRFFOBCJ3AGWC4K47/"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2019-6690",
        "dateUpdated": "2021-12-15",
        "state": "PUBLIC"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}